app/views/layouts/admin/_tables.html.slim

```diff
@@ -1,3 +1,4 @@
 - content_for :styles
   = stylesheet_link_tag "jquery-ui", :media => "all"
   = stylesheet_link_tag "vendor/wice_grid", :media => "all"
+  = stylesheet_link_tag "admin/table/table", :media => "all"
```

Comment:
I don't think you should need to include this explicitly. The Rails asset pipeline should pick it up.